uint8 node1
uint8 node2
float32 distance
